relnotes/client-name.feature.md

```diff
@@ -0,0 +1,8 @@
+## Name of connected client is now accessible via `Connection`
+
+`swarm.neo.node.Connection`
+
+The node `Connection` class has a new method -- `connected_client` -- that
+returns the name of the connected client received during authentication. This
+is useful for logging.
+
```

Comment:
I assume the commit order on the github is wrong (because this file in the last gh commit is changed in the first gh commit). Useful trick: `git rebase -i -x 'git commit --amend --date=$(date +%s) --no-edit' upstream/v4.x.x` will fix it. Not important for this PR, as it's trivial.

Another small point: we don't use indentation in the new release notes format anymore (IIRC we used to have an asterisk in front of the module names and the description would be indented by two spaces)?

Reply:
Yeah the commit order is displayed wrong.

>we don't use indentation in the new release notes format anymore?

Correct. We're using headings instead of a bullet list.

Reply:
So, this is wrong: https://github.com/sociomantic-tsunami/ocean/blob/186faa8572df58714fa54b0856a8ffcaf21b41d2/relnotes/baguette.migration.md ?

Reply:
According to https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-contributor.rst, yes.

Reply:
(Though it's only _slightly_ wrong.)

Reply:
Thanks!